### ISSUE TITLE
You need to update your example code

### ISSUE THREAD

**Issue by parkoon** [cuongdevjs/reactjs-social-login#130]
`access_type=offline` is added on 2.6.3
so, [your google login example code](https://github.com/cuongdevjs/reactjs-social-login/blob/master/example/src/App.tsx) is not working as expected.

**Comment by amin-da** [comment]
hi @cuongdevjs 
In the demo version of this library at the address below
https://react-social-login.netlify.app/
google login send me CORS error  and noting happening!
why?
google login is correct?

**Comment by cuongdevjs** [comment]
yes, cors sometimes happen when I try to get profile users, this process would be implemented on the server side instead. You won't see this if you only get the code or token and send it to the backend (use prop `isOnlyGetToken` or `isOnlyGetCode`)

**Comment by amin-da** [comment]
thanks for reply @cuongdevjs 
one another question
This library is good for production? And there is no problem with next js 12.5 ?

**Comment by cuongdevjs** [comment]
> thanks for reply @cuongdevjs one another question This library is good for production? And there is no problem with next js 12.5 ?

sure, I am using this library through multiple projects and it works well. the lib is absolutely compatible with the newest react version, but in NextJS, please use it with ssr = false, this library only works on the client side.

**Comment by Shams-02** [comment]
@cuongdevjs

![image](https://github.com/cuongdevjs/reactjs-social-login/assets/111973876/fd6a4187-64d6-4df4-8a41-4b48105f19d1)


Still Getting this error: Cross-Origin-Opener-Policy policy would block the window.closed call.


with this implementation:
` <LoginSocialGoogle
 isOnlyGetToken
client_id={""}
            onLoginStart={onLoginStart}
            onResolve={({ provider, data }: IResolveParams) => {
              setProvider(provider)
              setProfile(data)
            }}
            onReject={(err) => {
              console.log(err)
            }}
          >
            <GoogleLoginButton />
        </LoginSocialGoogle>`

**Comment by hassaan-glowfish** [comment]
having same issue
